src/main/java/com/rarchives/ripme/ui/MainWindow.java

```diff
@@ -946,7 +963,11 @@ public void mouseClicked(MouseEvent e) {
                     mainFrame.setAlwaysOnTop(false);
                 }
             });
-        } catch (Exception e) {
+        } catch (IOException e) {
+            //TODO implement proper stack trace handling this is really just intented as a placeholder until you implement proper error handling
+            e.printStackTrace();
+        } catch (AWTException e) {
+            //TODO implement proper stack trace handling this is really just intented as a placeholder until you implement proper error handling
```

Comment:
Yeah, that's true, but I think this code pattern is fair though. The point is to catch the exception, print it out so that someone can handle it from the log, and not actually crash the program. (It is possible to continue the program and just not finish the operation that excepted.)